src/definition/uikit/UIKitInteractionPayloadFormatter.ts

```diff
@@ -26,6 +26,31 @@ export function formatModalInteraction(view: IUIKitModalViewParam, context: IUIK
     };
 }
 
+export function formatContextualBarInteraction(view: IUIKitContextualBarViewParam, context: IUIKitInteraction): IUIKitContextualBarInteraction {
+    if (![UIKitInteractionType.CONTEXTUAL_BAR_OPEN,
+        UIKitInteractionType.CONTEXTUAL_BAR_UPDATE,
+        UIKitInteractionType.CONTEXTUAL_BAR_CLOSE].includes(context.type)) {
```

Comment:
If we use a type guard function to validated `context.type`, Typescript will be able to tell the actual type of the property